src/testHelpers.ts

```diff
@@ -0,0 +1,14 @@
+import { render, RenderOptions } from '@testing-library/react'
+
+export const renderWithModalRoot = (
```

Comment:
Use this when writing Jest tests for modal components (or components that render modals)